### PR TITLE
adding XRBLOCK metrics

### LINKED ISSUE #35: Importing metrics from the XRBlock discussions / docs to webrtc-stats.
The metrics are discussed in https://tools.ietf.org/html/draft-ietf-xrblock-rtcweb-rtcp-xr-metrics-03

### PR DESCRIPTION
**Pull request by vr000m**
Moved metrics from: https://tools.ietf.org/html/draft-ietf-xrblock-rtcweb-rtcp-xr-metrics
Fixes #35.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -503,6 +511,84 @@
...
+                  The fraction of RTP packets lost during bursts, Appendix A (a) of [[!RFC7004]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstDiscardRate</code></dfn> of type <span class=
+                  "idlMemberType"><a>double</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The fraction of RTP packets discarded during bursts, Appendix A (e) of [[!RFC7004]].
```
Comment (alvestrand) [inline webrtc-stats.html]: I hate rates, especially when it's obtained by dividing two numbers that are available. But the "packetsExpected" isn't available. Why not?

The 7004 definitions involves multiplying by 32768. We shouldn't do that.

File: webrtc-stats.html [hunk 2]
```diff
@@ -829,11 +916,21 @@
...
+                  <dfn><code>framesLost</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                    Only valid for video. <code>framesLost</code> is the cumulative number of
+                    full frames lost, as defined in Appendix A (i) of [[!RFC7004]].
+                  </p>
+                </dd>
+                <dt>
```
Comment (alvestrand) [inline webrtc-stats.html]: Why is "full frames lost" included, and "partial lost frames" is not?

File: webrtc-stats.html (hunk 1 above)
Comment (vr000m) [inline webrtc-stats.html]: Will make a note on keeping the actual values.

File: webrtc-stats.html (hunk 2 above)
Comment (vr000m) [inline webrtc-stats.html]: The partial lost frames show up as corrupted frames. However, I can add the partial frames lost for applications to make analysis of why the corruption happened.

Comment (alvestrand) [inline webrtc-stats.html]: ... because "corrupted" is the same number. Duh. Add 7004 as defiition for "corrupted"?

File: webrtc-stats.html (hunk 1 above)
Comment (vr000m) [inline webrtc-stats.html]: @alvestrand the packetsExpected is known by the RTP stack. It is already used in fractionLost and is based on RFC3550. see https://tools.ietf.org/html/rfc3550#appendix-A.3

**Comment by vr000m** [comment]
@alvestrand PTAL. I added partialFramesLost. However, I cannot really think of another reason for frame corruption.

**Comment by vr000m** [comment]
@alvestrand could you review? feedback appreciated.

File: webrtc-stats.html [hunk 3]
```diff
@@ -503,6 +511,96 @@
...
+                  <dfn><code>burstLossCount</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of bursts of lost RTP packets,
+                  Appendix A (e) of [[!RFC6958]].
+                  </p>
+                  <p>[[!RFC3611]] recommends a Gmin (threshold) value of
+                  16 for classifying packet loss or discard burst.
```
Comment (alvestrand) [inline webrtc-stats.html]: should this be "for classifying a sequence of packet losses or discards as a burst"?

File: webrtc-stats.html [hunk 4]
```diff
@@ -503,6 +511,96 @@
...
+                <dt>
+                  <dfn><code>burstLossRate</code></dfn> of type <span class=
+                  "idlMemberType"><a>double</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  It is the fraction of RTP packets lost during bursts to the total
+                  number of RTP packets expected in the bursts. As defined in
+                  Appendix A (a) of [[!RFC7004]], however, the actual value is
+                  reported without multiplying with 32768.
```
Comment (alvestrand) [inline webrtc-stats.html]: nit: "without multiplying by" is better English, I think.

File: webrtc-stats.html [hunk 5]
```diff
@@ -503,6 +511,96 @@
...
+                  Appendix A (e) of [[!burst-gap-discard]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstLossRate</code></dfn> of type <span class=
+                  "idlMemberType"><a>double</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  It is the fraction of RTP packets lost during bursts to the total
```
Comment (alvestrand) [inline webrtc-stats.html]: For consistency, don't start the definition with "it is".

**Comment by alvestrand** [comment]
looks good, modulo a couple of nits.

File: webrtc-stats.html (hunk 5 above)
Comment (vr000m) [inline webrtc-stats.html]: Just for this section, because the document has several definitions starting with "It is".

**Comment by vr000m** [comment]
Merging the PR. If there are new issues, we can do it in a separate PR.

**Pull request merged**